modules/rostests/rosautotest/main.cpp

```diff
@@ -100,6 +110,22 @@ wmain(int argc, wchar_t* argv[])
             WineTest.Run();
         }
 
+        /* Clear the stringstream */
+        ss.str("");
+        ss.clear();
+
+        /* Show the beginning time */
```

Comment:
maybe "Show again the beginning time" ; with "again" implicitly referring to the fact that it was already shown somewhere else (i.e. when starting the tests).